cityofaustin/atd-moped
# Add secondary name to activity log

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/16650

## Testing
**URL to test:** 

test locally

**Steps to test:**

add a new secondary name to a project that currently doesnt have one. the activity log should say the secondary name has been changed. 

remove the secondary name, but saving an empty string. the activity log should say the secondary name was changed to (none)


---
#### Ship list
